rpcs3/Emu/Cell/SPUCommonRecompiler.cpp

```diff
@@ -543,12 +543,33 @@ extern void utilize_spu_data_segment(u32 vaddr, const void* ls_data_vaddr, u32 s
 		return;
 	}
 
-	for (u32 addr : obj.funcs)
+	if (spu_log.notice)
```

Comment:
Isn't this missing the check?
```suggestion
	if (spu_log.notice && !obj.funcs.empty())
```

Reply:
It's checked above actually

Reply:
lol